src/iccpd/src/iccp_cmd_show.c

```diff
@@ -94,16 +94,20 @@ int iccp_mclag_config_dump(char * *buf,  int *num, int mclag_id)
         memcpy(state_info.peer_ip, csm->peer_ip, ICCP_MAX_IP_STR_LEN);
 
         if (peer_link_if)
+        {
             memcpy(state_info.peer_link_if, peer_link_if->name, ICCP_MAX_PORT_NAME);
+            memcpy(state_info.peer_link_mac, peer_link_if->mac_addr, ETHER_ADDR_LEN);
```

Comment:
Is this linbe trying to fix something that was not mentioned in your PR??  Did someone reviewed this change?  The other changes looks straight forward but not this one...

Reply:
1) the magic number 6 was replaced by the appropriate constant instead.
2) there was a confusing logic with duplicated check.
```
if (peer_link_if) 
    do_some_stuff()
else
    do_another_stuff()

if (peer_link_if)
    do_the_rest()

```
As those lines needed to be changed in this PR simplified it.

Reply:
Oh... I overlooked the original code that you changed from.  Must have missed my coffee that day.  My bad...  It makes perfect sense for this change.